build_test/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.10.7+3
+
+- Handle the case where the root package in a `PackageAssetReader` is a fake
```

Comment:
When does that happen? Just curious...

Reply:
`PackageAssetReader` is designed to let you read assets from real packages imported by the current isolate. We combine it with asset readers for in-memory assets that have fake packages. With things like [`resolveSource`](https://github.com/dart-lang/build/blob/b3c63cc5ed1b6993b557f6f2faf4d153b6979cc2/build_test/lib/src/resolve_source.dart#L27) we pass the [`rootPackage`](https://github.com/dart-lang/build/blob/b3c63cc5ed1b6993b557f6f2faf4d153b6979cc2/build_test/lib/src/resolve_source.dart#L177) as one of those fake ones.

Reply:
Ah! Okay. Still zero idea. That's fine. I trust you. 😁